edn_format/edn_parse.py

```diff
@@ -142,9 +183,13 @@ def p_expression_tagged_element(p):
     p[0] = output
 
 
+def eof():
+    raise SyntaxError('EOF Reached')
```

Comment:
Should this be an `EDNDecodeError` exception as in the other PR?

Reply:
Yes!